test/test_cli.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2016  Red Hat, Inc
```

Comment:
We talked after the new year about updating the copyrights.  This is why I'd kinda prefer doing them all at once.